src/views/HomePage.vue

```diff
@@ -2,241 +2,280 @@
-              button-label="usage.components.actionBar.buttons.getText.title"
-              :icon="create"
-              @click="openTextInputModal()"
-            />
-            <ms-action-bar-button
-              id="action-bar-button-example"
-              button-label="usage.components.actionBar.buttons.getPassword.title"
-              :icon="lockClosed"
-              @click="openPasswordInputModal()"
-            />
-            <ms-action-bar-button
-              id="action-bar-button-example"
-              button-label="usage.components.actionBar.buttons.settings.title"
-              :icon="cog"
-              @click="openSettingsModal()"
-            />
-            <ms-action-bar-button
-              id="action-bar-button-example"
-              button-label="usage.components.actionBar.buttons.importDocument.title"
-              :image="DocumentImport"
+          <div class="example-divider-content">
+            <ms-action-bar id="ms-action-bar-example">
+              <ms-action-bar-button
+                id="action-bar-button-example"
+                button-label="usage.components.actionBar.buttons.alert.title"
+                :icon="warning"
+                @click="openAlertModal()"
+              />
+              <ms-action-bar-button
+                id="action-bar-button-example"
+                button-label="usage.components.actionBar.buttons.askQuestion.title"
+                :icon="helpCircle"
+                @click="openQuestionModal()"
+              />
+              <ms-action-bar-button
+                id="action-bar-button-example"
+                button-label="usage.components.actionBar.buttons.getText.title"
+                :icon="create"
+                @click="openTextInputModal()"
+              />
+              <ms-action-bar-button
+                id="action-bar-button-example"
+                button-label="usage.components.actionBar.buttons.getPassword.title"
+                :icon="lockClosed"
+                @click="openPasswordInputModal()"
+              />
+              <ms-action-bar-button
+                id="action-bar-button-example"
+                button-label="usage.components.actionBar.buttons.settings.title"
+                :icon="cog"
+                @click="openSettingsModal()"
+              />
+              <ms-action-bar-button
+                id="action-bar-button-example"
+                button-label="usage.components.actionBar.buttons.importDocument.title"
+                :image="DocumentImport"
+              />
+            </ms-action-bar>
+          </div>
+        </ion-item-divider>
+
+        <!-- progress bar -->
+        <ion-item-divider class="example-divider">
+          <ion-label class="title-h2">{{ $msTranslate('usage.components.progressBar.title') }}</ion-label>
+          <div class="example-divider-content">
+            <ms-progress-bar
+              :progression="'79%'"
+              :width="'16rem'"
```

Comment:
You don't need the `:` if you just assign a string, you're forcing yourself to add simple quotes here.

```suggestion
              progression="79%"
              width="16rem"
```